rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -824,6 +828,81 @@ void getFederationElection_whenElectionIsNotInStorage_shouldReturnDefault() {
 
     }
 
+    @Test
+    void getNextFederationCreationBlockHeight_preIris300_storageIsNotAccessedAndReturnsEmpty() {
```

Comment:
```suggestion
    void getNextFederationCreationBlockHeight_whenPapyrusIsActive_storageIsNotAccessedAndReturnsEmpty() {
```

Reply:
`whenPapyrusIsActive` is ambiguous. When papyrus is active, any of the other forks after it could be active as well, iris, hop, etc.
And since the logic being tested where introduced in iris, I think it makes sense to keep it like "pre iris" and "post iris".

Reply:
Well, I've been using that convention, and I guess I have seen others using it. Also, I believe it is ok to assume that no hard fork after it is active if the test is written using [ActivationConfigsForTest](https://github.com/rsksmart/rskj/blob/master/rskj-core/src/test/java/org/ethereum/config/blockchain/upgrades/ActivationConfigsForTest.java) class due to the way that class works.